src/sentry/tasks/delete_seer_grouping_records.py

```diff
@@ -16,15 +16,15 @@
 from sentry.taskworker.namespaces import seer_tasks
 from sentry.utils.query import RangeQuerySetWrapper
 
-BATCH_SIZE = 1000
+BATCH_SIZE = 100
 
 logger = logging.getLogger(__name__)
 
 
 @instrumented_task(
     name="sentry.tasks.delete_seer_grouping_records_by_hash",
     queue="delete_seer_grouping_records_by_hash",
-    max_retries=0,
+    max_retries=0,  # XXX: Why do we not retry?
```

Comment:
I'm a bit shocked we don't retry. Any thoughts as to why we shouldn't?

Reply:
You could add a retry handler. Make sure you allow list one or more exception types though. Without that retries don't happen unless a `TimeoutError` is raised.